how-tos/regex/README.md

```diff
@@ -44,4 +42,9 @@ _environment.push(
         "_active" : true
     }
 )
-```
\ No newline at end of file
+```
+If you check the logs you can see that IDs start with something like `environmentName.[0]Environment` or `GameCore`.<br>
+This part of the ID is unnecessary and you can get rid of it.<br>
```

Comment:
Please explain why this is a good idea. For example:

###
Since the length of the regex has some impact on performance, consider only matching on the last unique section of the ID that you are interested in.<br>
###

That clarifies the "why", without leading people to believe that the examples given below are equivalent in their "before" and "after".

Reply:
`Since the length of the regex has some impact on performance..`

This is the reason I wrote the example for the BTS `PillarPair` stuff the way I did. Performance with LookupID / Hecks regex validator can be horrendous.

Reply:
It really depends on input data too. If there's going to be many long partial (but not full) matches on the start of the regex, performance will suffer a lot more, which I imagine is the case here.

Reply:
Response from Tzur on Discord:

```
Long regex is more specific so smaller loading times but at the cost of performance.
We can see that all the IDs start in a similar way it will not save time and will just affect performance.